package.json

```diff
@@ -29,11 +29,11 @@
   "dependencies": {
     "@babel/core": "^7.17.5",
     "@babel/plugin-proposal-decorators": "^7.17.2",
-    "rollup": "^2.61.1",
-    "rollup-plugin-sourcemaps": "^0.6.3",
     "@rollup/plugin-babel": "^5.3.1",
-    "@rollup/plugin-commonjs": "^21.0.1",
+    "@rollup/plugin-commonjs": "^22.0.0",
```

Comment:
This change helps to fix `status-message-collections`, but I have no idea why :/

Reply:
Maybe some bug fixes happened in 22.0?